### PR TITLE
Add documentation for hardware setup 

### LINKED ISSUE #46: README: start driver node, then start URCap
Document how to run with hardware

### PR DESCRIPTION
**Pull request by Abishalini**
Resolves #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,18 @@ Beta version of the ROS2 Universal Robots driver. Should be transferred to the U
 **NOTE**: If you are a ROS2 beginner and get lost consult the [references - section](#references).
 
 
+### How to Setup the UR Robot
```
Comment (AndyZe) [inline README.md:39]: Please use 2X ## like the sections above and below

**Review by AndyZe — APPROVED** [review]
Thanks! This is really important to have.

Can you clean up some other little things while we're here:

- L122, replace "5." with a dash "-"
- L122, "test the description"

File: README.md [hunk 2]
```diff
@@ -36,6 +36,18 @@ Beta version of the ROS2 Universal Robots driver. Should be transferred to the U
...
+
+1. To enable external control of a UR robot from a remote PC, [install URCap](/ur_robot_driver/doc/install_urcap_e_series.md).
+
+2. Switch to Remote Control by clicking on the teach pendant logo on the upper right corner.
+
+3. Create a new program. In the Installation tab of the teach pendant, select the UPCaps option on the left and adjust the IP address and port number. The remote host's IP can be found using `ifconfig`.
```
Comment (nbbrooks) [inline README.md:45]:
```suggestion
3. Create a new program. In the Installation tab of the teach pendant, select the URCaps option on the left and adjust the IP address and port number. The remote host's IP can be found using `ifconfig`.
```

URCaps?

**Review by nbbrooks — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (nbbrooks) [reply README.md:45]: For formatting suggestions I have found it least confusing to simply do "Insert a suggestion."

File: README.md (hunk 1 above)
Comment (nbbrooks) [reply README.md:39]: For formatting suggestions I have found it least confusing to simply do "Insert a suggestion."

**Review by destogl — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -36,6 +36,18 @@ Beta version of the ROS2 Universal Robots driver. Should be transferred to the U
 **NOTE**: If you are a ROS2 beginner and get lost consult the [references - section](#references).
 
 
+## How to Setup the UR Robot
+
+1. To enable external control of a UR robot from a remote PC, [install URCap](/ur_robot_driver/doc/install_urcap_e_series.md).
+
+2. Switch to Remote Control by clicking on the teach pendant logo on the upper right corner.
```
Comment (destogl) [inline README.md:43]: This seems not to be right. @MGBla can you please check/comment?

```suggestion
```

**Comment by AndyZe** [comment]
Let's get this merged so users have some guidance of how to use the hardware. @MGBla please open an issue after checking what Denis mentioned above.

**Review by nbbrooks — APPROVED** [review]

**Pull request merged**